### ISSUE TITLE
cant find AdaptiveWingLoss implement

### ISSUE THREAD

**Issue by dtiny** [protossw512/AdaptiveWingLoss#10]
I check this repository and dont find any file or function implement Adaptive Wing loss.
Could you sure AdWingloss shared.

**Comment by protossw512** [comment]
@dtiny The AWing loss implementation will be uploaded with the training code.

**Comment by joydeepmedhi** [comment]
Hi,
Please provide the loss implementation.

Thanks.

**Comment by houlin** [comment]
已经2021年啦，食言拉吧

**Comment by geekac** [comment]
2022年3月16日14:01:04
